postings/vm_software_engineer.md

```diff
@@ -179,37 +179,37 @@ CyberSkill codes:
-[T0054]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/tasks?id=T0054
-[T0207]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/tasks?id=T0207
-[T0418]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/tasks?id=T0418
-[T0498]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/tasks?id=T0498
-[T0501]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/tasks?id=T0501
-[T0569]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/tasks?id=T0569
-[T0585]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/tasks?id=T0585
-[T0615]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/tasks?id=T0615
-[T0758]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/tasks?id=T0758
-[T0761]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/tasks?id=T0761
-[T0783]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/tasks?id=T0783
-[T0786]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/tasks?id=T0786
-[T0834]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/tasks?id=T0834
-[K0140]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/knowledge?id=K0140
-[K0001]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/knowledge?id=K0001
-[K0202]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/knowledge?id=K0202
-[K0332]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/knowledge?id=K0332
-[S0149]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/skills?id=S0149
-[A0047]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/abilities?id=A0047
-[K0499]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/knowledge?id=K0499
-[K0565]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/knowledge?id=K0565
-[K0610]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/knowledge?id=K0610
-[S0211]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/skills?id=S0211
-[S0218]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/skills?id=S0218
-[S0080]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/skills?id=S0080
-[K0130]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/knowledge?id=K0130
-[K0346]: https://niccs.cisa.gov/workforce-development/cyber-security-workforce-framework/knowledge?id=K0346
+[SP-DEV-001]: https://niccs.cisa.gov/workforce-development/nice-framework/workroles?name=All&id=SP-DEV-001
```

Comment:
Pedantic but if you wouldn't mind (this can be ignored if time critical or strongly opposed) would you sort these alphabetically?

Reply:
Yup yup.  np

Reply:
[6ee0ba2](https://github.com/cisagov/join-cisagov/pull/16/commits/6ee0ba246d052651e271ffd05549ce735a0b4ad6)